### ISSUE TITLE
USWDS-Site - Bug: Touchpoints referrer is not being captured

### ISSUE THREAD

**Issue by bonnieAcameron** [uswds/uswds-site#1706]
### Describe the bug

Sometime in late April, the Touchpoints widget stopped capturing the referrer when end users submit feedback on particular pages. This results in us receiving high quality, actionable feedback, but not being able to discern where exactly the fix needs to be applied in many cases. 

### Expected Behavior

Each time someone submits feedback, the Touchpoints app should capture and report on which page the feedback was submitted. 

### Additional context

Steps to fix: 

Reach out to Touchpoints support to see if they know how to fix this issue. May create other tickets as a result to implement the fix.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.

**Comment by bonnieAcameron** [comment]
Contact from Touchpoints is taking a look

**Comment by bonnieAcameron** [comment]
From Ryan Wold:

"I've spent some time looking at this, and can't nail this down.
I'm looking at how different browsers treat the referrer attribute.

Ultimately, I browse from a the USWDS site to the Touchpoints site, and when I arrive, the `document.referrer` is only the home path.
I did see one response a week ago that had a different path 🤔. (also had an old/obscure browser)."

I've asked Ryan what he recommends as next steps for us to take.
